nhais-adaptor/adaptor/outgoing/death/message_death_adaptor.py

```diff
@@ -0,0 +1,51 @@
+import adaptor.fhir_helpers.fhir_finders as finders
+from adaptor.fhir_helpers.fhir_creators import ResourceType, ParameterName
+from adaptor.outgoing.message_adaptor import MessageAdaptor
+from edifact.outgoing.models.death.message_death import MessageSegmentDeathPatientDetails, \
+    MessageSegmentDeathRegistrationDetails
+
+
+class MessageDeathAdaptor(MessageAdaptor):
+
+    def get_reference_number(self) -> str:
+        """
+        Returns the unique reference number to indicate a death registration in the edifact payload
+        :return: a string representing a death registration
+        """
+        return "G5"

```

Comment:
Is there any particular reason a method over a variable?

Reply:
Nevermind, I can see it's an inherited method from a general method in the MessageAdaptor, all good.

Reply:
👍